neo/Network/RPC/Model/RpcBlock.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Neo.IO.Json;
+using Neo.Network.P2P.Payloads;
+
+namespace Neo.Network.RPC.Model
```

Comment:
```suggestion
namespace Neo.Network.RPC.Models
```

And many other classes are the same.